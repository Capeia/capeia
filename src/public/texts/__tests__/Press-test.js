// @flow
/* eslint-env jest, jasmine */

import React from 'react'
import renderer from 'react-test-renderer'

describe('<Press>', () => {
  it('renders correctly', () => {
    jest.deepUnmock('public/texts/Press')
    const Component = require('public/texts/Press').default
    const tree = renderer.create(<Component />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
