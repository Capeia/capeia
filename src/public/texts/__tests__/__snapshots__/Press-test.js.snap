// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Press> renders correctly 1`] = `
<div
  className="container-fluid"
>
  <div
    className="row"
  >
    <div
      className="contentContainer  col-lg-6 col-lg-offset-3 col-md-8"
    >
      <div
        className={undefined}
      >
        (Closed Beta)
      </div>
    </div>
    <div
      className="sidebarContainer col-lg-3 col-md-4"
    >
      <aside
        className="sidebar"
      />
    </div>
  </div>
</div>
`;
